api/src/main/java/uk/gov/hmcts/payment/api/domain/service/ServiceRequestDomainServiceImpl.java

```diff
@@ -255,4 +268,23 @@ public ResponseEntity createIdempotencyRecord(ObjectMapper objectMapper, String
     public Boolean isDuplicate(String orderReference) {
         return Optional.of(find(orderReference)).isPresent();
     }
+
+    @Override
+    public void sendMessageTopicCPO(ServiceRequestCpoDto serviceRequestDto) {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            Message msg = new Message(objectMapper.writeValueAsString(serviceRequestDto));
+
+            msg.setContentType("application/json");
+            msg.setLabel("Service Callback Message");
+            msg.setProperties(Collections.singletonMap("serviceCallbackUrl",
+                "https://cpo-case-payment-orders-api-demo.service.core-compute-demo.internal/case-payment-orders"));
+
+            TopicClientProxy topicClientCPO = new TopicClientProxy(connectionString.trim(), topic);
+            topicClientCPO.send(msg);
+
+        } catch (Exception e) {
+            LOG.error("Error", e);
```

Comment:
Please, send back valid error to client, and also check transactional rollback will work on error